config-windows.md

```diff
@@ -0,0 +1,99 @@
+# Windows-specific Container Configuration
+
+This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
+
+## Resources
+
+You can configure a container's resource limits via the OPTIONAL `resources` field of the Windows configuration.
+
+### Memory
+
+`memory` is an OPTIONAL configuration for the container's memory usage.
+
+The following parameters can be specified:
+
+* **`limit`** *(uint64, optional)* - sets limit of memory usage in bytes.
```

Comment:
Yes. More information on overall memory management of containers on Windows (which is different between Windows Server and Hyper-V containers) will eventually make it into the documentation on microsoft.com, but for now there's nothing published as such.